AmplifyPlugins.podspec

```diff
@@ -46,8 +46,9 @@ Pod::Spec.new do |s|
 
     # This is technically redundant, but adding it here allows Xcode to find it
     # during initial indexing and prevent build errors after a fresh install
-    s.dependency 'AWSAuthCore', $OPTIMISTIC_AWS_SDK_VERSION
-
+    ss.dependency 'AWSAuthCore', $OPTIMISTIC_AWS_SDK_VERSION
+    ss.dependency 'AWSCognitoIdentityProvider', $OPTIMISTIC_AWS_SDK_VERSION
+    ss.dependency 'AWSCognitoIdentityProviderASF', '1.0.2'
```

Comment:
Seems that `AWSCognitoIdentityProviderASF` is currently `2.16.0` in the develop branch... and it was previously 1.1.0?  Do we want to adopt the 2.x.y scheme, or...?

Note looks like the Podfile.lock files also got updated?

Reply:
Locks got updated because I upgraded to 2.16. The ASF version doesn't start conforming with the overall SDK version until 2.17.0.